addon/components/accordion-list.js

```diff
@@ -13,11 +13,21 @@ const AccordionListComponent = Ember.Component.extend({
 
   // Internal state
   _activeItems: null,
+  _registeredItems: null,
   init() {
     this._super(...arguments);
     this.set('_activeItems', Ember.A([]));
+    this.set('_registeredItems', Ember.A([]));
   },
 
+  // If each registered item has at least on panel open, then return true
+  allExpanded: computed('_activeItems.[]', '_registeredItems.[]', function() {
+    let { _activeItems, _registeredItems } = this.getProperties('_activeItems', '_registeredItems');
+
+    let anyMissing = _registeredItems.reduce((anyMissing, item) => anyMissing || isNone(_activeItems.findBy('id', item.itemId)), false);
```

Comment:
`Array.some()` will short-circuit completion, and you don't have to deal with the accumulator